apps/io500/readme.md

```diff
@@ -0,0 +1,29 @@
+# IO-500 HPC Storage benchmark
+
+The [IO-500](https://www.vi4io.org/std/io500/start) is a I/O storage benchmark designed to measure and compare different HPC storage solutions. It runs a suite of throughput, IOPS and metadata benchmarks and gives a score to rank the storage solution. It uses modified versions of LLNL IOR/MDtest and a find benchmark.
+
+The IO-500 benchmark needs to be built on HB or HC sku's with CentOS-HPC 7.6 (using mpi/mpich-3.3). See one of the examples for building a Cluster with HB or HC skus and PBS. (e.g. [simple_hpc_pbs](../../examples/simple_hpc_pbs/readme.md))
+
+log-on to the headnode
+```
+azhpc-connect headnode
+```
+
+Copy apps dir to the headnode
+```
+azhpc-scp -r $azhpc_dir/apps hpcuser@headnode:.
+```
+
+Build the IO-500 benchmark suite from the build script.  This will install IO-500 into /apps/io-500-dev and create a modulefile:
+```
+    build_io500.sh
```

Comment:
you should replace that by a azhpc-run command. The steps would be :

- azhpc-scp
- azhpc-run
- azhpc connect
and then submit jobs

Reply:
I think its more natural to install software and run jobs from headnode (after logging into headnode).
If there is a strong preference to use azhpc-scp instead I can make that change.

Reply:
both works, this is not a blocker, just aligned with what we are doing with other apps.